src/Project/HCA/serialization.content/Content/HCA/HCA Email.yml

```diff
@@ -0,0 +1,35 @@
+﻿---
+ID: "4ed70d45-b821-461d-bba8-3407b7b1d48e"
+Parent: "bbf003f2-f4d3-415f-8136-357b71508316"
+Template: "cf9c8a2a-2794-4fea-980a-ef8426f3d6c3"
+Path: /sitecore/content/HCA/HCA Email
+DB: master
+BranchID: "f8fae92a-4f10-470c-ad5c-7ed9e6b636af"
+SharedFields:
+- ID: "1b963507-6176-4336-a14d-d5070c3b0286"
+  Hint: Base URL
+  Value: "http://hca.local"
+- ID: "52d572ad-9678-423f-96e2-477528418ba1"
+  Hint: "Global Opt-out List"
+  Value: "d03c9cca-98f9-41fe-a97d-9ff33aae2b74"
+- ID: "826730f6-4aeb-4fa5-a55b-b02fece73c14"
+  Hint: Brand Name
+  Value: HCA
+- ID: "dec8d2d5-e3cf-48b6-a653-8e69e2716641"
+  Hint: __Security
+  Value: |
+    ar|sitecore\EXM Users|pd|^*|+item:read|pe|^*|+item:read|ar|sitecore\EXM Advanced Users|pd|+item:delete|+item:admin|+item:create|+item:rename|^*|+item:read|+item:write|pe|+item:delete|+item:admin|+item:create|+item:rename|^*|+item:read|+item:write|
+- ID: "f6d8a61c-2f84-4401-bd24-52d2068172bc"
+  Hint: __Originator
+  Value: "{DB7A8703-C5AE-41FF-8B23-264195634B9E}"
+Languages:
+- Language: en
+  Versions:
+  - Version: 1
+    Fields:
+    - ID: "0f383cd8-c031-4a9d-9ed0-d55217f91287"
+      Hint: From Address
+      Value: hca@devmail.epam.com
```

Comment:
Also here. Can we change it to "no-reply@devmail.epam.com"

Reply:
Yes, we can